src/Sulu/Bundle/ContactBundle/Resources/js/components/ContactDetails/ContactDetails.js

```diff
@@ -160,9 +160,8 @@ class ContactDetails extends React.Component<Props> {
         const {value: {emails}} = this.props;
 
         if (emails.length === 0) {
-            // $FlowFixMe
             return [
-                this.getEmptyEntry('emails'),
+                (this.getEmptyEntry('emails'): any),
```

Comment:
I would not just cast it to `any`, since we are losing potential subsequent type checks. If we do something like this, I would cast to the type we want afterwards as well:

```javascript
((this...): any): EmailType)
```